### PR TITLE
F/file mover

### PR DESCRIPTION
**Pull request by jmagoon**
- make file moving work with `Any` params
- moves from `/files` in the container to the workdir so we don't end up re-taring everything every time
- makes the rebuild button work because
- uses gpt-4o

### CODE REVIEW

**Review by plparent — APPROVED** [review]
The go code looks good to me.

File: frontend/agents/gpt-4_python_compute/generate/computations.py [hunk 1]
```diff
@@ -109,7 +109,7 @@ def compute(user_prompt, model_version, conversation_history, apiKey):
         apiKey = data["apiKey"]
 
         # Call the compute function and get the result
-        result = compute(user_prompt, 'gpt-4', conversation_history, apiKey)
+        result = compute(user_prompt, 'gpt-4o', conversation_history, apiKey)
```
Comment (FGRCL) [inline frontend/agents/gpt-4_python_compute/generate/computations.py:112]: There's no need to update the langchain version for this? We also might want to upgrade it for the view agent as well.

File: frontend/src/components/ui/RunPipelineButton.jsx [hunk 2]
```diff
@@ -75,9 +75,11 @@ export default function RunPipelineButton({modalPopper, children, action}) {
       pipelineSpecs['build'] = pipeline.buffer
       pipelineSpecs['name'] = pipeline.name
       pipelineSpecs['id'] = pipeline.id
+      const rebuild = (action == "Rebuild")
       const execution = {
         id: executionId,
         pipeline: pipelineSpecs,
+        build: rebuild
```
Comment (FGRCL) [inline frontend/src/components/ui/RunPipelineButton.jsx:82]: So the rebuild button also triggers a run?

**Review by FGRCL — APPROVED** [review]
Works on my end

File: frontend/src/components/ui/RunPipelineButton.jsx (hunk 2 above)
Comment (jmagoon) [reply frontend/src/components/ui/RunPipelineButton.jsx:82]: Yep

File: frontend/agents/gpt-4_python_compute/generate/computations.py (hunk 1 above)
Comment (jmagoon) [reply frontend/agents/gpt-4_python_compute/generate/computations.py:112]: Just changes the model in the api call. Added to view

**Pull request merged**